pom.xml

```diff
@@ -2042,6 +2048,31 @@
                <artifactId>protocol-parser-generator</artifactId>
                <version>${version.infinispan.maven-plugins}</version>
             </plugin>
+            <plugin>
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-project-info-reports-plugin</artifactId>
+               <version>3.1.1</version>
+            </plugin>
+            <plugin>
+               <groupId>com.github.spotbugs</groupId>
+               <artifactId>spotbugs-maven-plugin</artifactId>
+               <version>4.0.4</version>
+            </plugin>
+            <plugin>
+               <groupId>org.owasp</groupId>
+               <artifactId>dependency-check-maven</artifactId>
+               <version>${version.owasp-dependency-check-plugin}</version>
+               <executions>
+                  <execution>
+                     <goals>
+                        <goal>aggregate</goal>
+                     </goals>
+                  </execution>
+               </executions>
+               <configuration>
+                  <suppressionFile>${infinispan.root}/etc/suppressions.xml</suppressionFile>
```

Comment:
I'd put `dependency-check` in the file name.

Reply:
Done